### DISCUSSION TITLE
Welcome to Stacks 2 Discussions!

### DISCUSSION THREAD

**Discussion by synapse** [stacks-task-manager/stacks2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

There are more ways in which you can get involved and contribute. Read more about it [here](https://getstacksapp.com/contributing).

**Comment by sarraro** [comment]
Hello from Italy, I am in this community because I consider stacks an intelligent and well-made application. I would like to see other features developed that would make it even more interesting and essential

**Reply by synapse** [reply]
Hey @sarraro Cris here from Stacks. What features would you have in mind?
Any idea is a good idea so feel free to share them.

**Comment by sarraro** [comment]
What an honour, the boss is writing to me!
Give me the possibility to have a printable report of the tasks in any exportable format. The programme needs this features to match up to expectations. Thank you
Roberto

**Reply by synapse** [reply]
@sarraro It's already planned for the final release 😉

**Reply by sarraro** [reply]
Perfect! I want to be the first to test it on the Mac ( if I can import my tasks from version 1 to version 2)
